dominode-bootstrapper/dominode_bootstrapper/geonodeadmin.py

```diff
@@ -181,22 +183,35 @@ def from_geonode_manager(cls, geonode_manager: GeoNodeManager):
         )
 
     def list_workspaces(self):
+        self.client.headers = self.headers
+
         response = self.client.get(
             f'{self.base_url}/rest/workspaces',
             auth=(self.username, self.password)
         )
+        response.raise_for_status()
         return response.json()
 
     def create_workspace(self, name):
+        self.client.headers = self.headers
+
         response = self.client.post(
             f'{self.base_url}/rest/workspaces',
             auth=(self.username, self.password),
-            json={"name": name}
+            json={
+                    "workspace": {
+                        "name": name
+                    }
+                }
         )
-        return response.json()
+        if response.status_code != 201:
+            return False
+        return True
```

Comment:
Keep consistency with the rest of the code. Instead of this pattern where you check the status code of the response manually and return a boolean, use `response.raise_for_status()` instead and raise an exception